packages/edit-site/src/hooks/commands/use-common-commands.js

```diff
@@ -288,6 +310,91 @@ function useGlobalStylesOpenRevisionsCommands() {
 		isEditorPage,
 		getCanvasMode,
 		setCanvasMode,
+		closeGeneralSidebar,
```

Comment:
You will also need to adjust the dependencies for the `useMemo` hook. Your code editor should tell you what dependencies you need and don't need.